src/core/defaultOptions.ts

```
@@ -0,0 +1,130 @@
+import type { ApolloClient } from "@apollo/client";
+import type { Prettify } from "@apollo/client/utilities/internal";
+
+export declare namespace DeclareDefaultOptions {
+  export interface WatchQuery {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)